### ISSUE TITLE
[FEA] Add String to ASCII characters support

### ISSUE THREAD

**Issue by bsuryadevara** [rapidsai/cudf#2622]
Right now there is no real way to change over string to ascii characters on cuDF dataframe column. Adding this feature would extremely helpful for generating vectorized sequences for a given string.

CPU:
```python
domains = ["nvidia.com", "google.com"]
def str2ascii_arr(domain):
     arr = [ord(c) for c in domain]
     return arr
sequence = [str2ascii_arr(domain) for domain in domains]
print(sequence)
```
Output:
```python
[[110, 118, 105, 100, 105, 97, 46, 99, 111, 109], [103, 111, 111, 103, 108, 101, 46, 99, 111, 109]]
```

**Comment by BartleyR** [comment]
If there's any way to get this ahead of GTC DC, one of our example workflows (DGA detection) is currently blocked by this issue. Or if we have a plausible workaround.

**Comment by davidwendt** [comment]
I created a prototype on `nvtext` the does the following:
```
import nvstrings, nvtext
from librmm_cffi import librmm
import numpy as np

s = nvstrings.to_device(['zyx','déf','12'])
arr = np.arange(8,dtype=np.int32)
d_arr = librmm.to_device(arr)

nvtext.code_points(s,d_arr.device_ctypes_pointer.value)

print(d_arr.copy_to_host())
```
```
[  122   121   120   100 50089   102    49    50]
```
The `s.len()` method can be used determine the length of each 'subarray'
```
>>> print(s.len())
[3, 3, 2]
```
These are all integers at this point so there may be other operations we can use to convert to offsets and/or pad the results appropriately.

**Comment by bsuryadevara** [comment]
@davidwendt thank you for the prototype. I will try if this fits to the process which requires str2ascii conversion.

**Comment by bsuryadevara** [comment]
@davidwendt when can we expect `code_points` feature?

**Comment by randerzander** [comment]
Closed by #2836
